kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-emf/src/main/java/org/eclipse/emf/ecore/xmi/resource/xml/XMLLoad.java

```diff
@@ -262,7 +262,7 @@ protected void traverse(Node node, AttributesProxy attributesProxy, XMLHandler h
                 break;
             }
             case Node.COMMENT_NODE: {
```

Comment:
Would be nice to add a test case for this case.

Reply:
Hi @domhanak will add.